docs/pages/tutorials/minimal/extend_schema.mdx

```diff
@@ -0,0 +1,329 @@
+import { CollapseCode } from "../../../components/CollapseCode";
+
+# Extend the schema
```

Comment:
"Add a table" might be a better name here

A table consists of a key schema and value schema, so "extend the schema" makes me think we'd be adding fields to the schema, not adding a new table.